src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -2414,6 +2414,101 @@ public GCCondemnedReasons[] PerHeapCondemnedReasons
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            MarkShortWeak,
+
+            /// <summary>
+            /// This field records the time spent for scanning the finalization queue and remarking the secondaries for dependent handles.
+            /// This happens for any type of GC, and it should not be zero for all GCs.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            MarkScanFinalization,
+
+            /// <summary>
+            /// This field records the time spent for scanning the long weak references
+            /// Long weak references are nulled out only if absolutely nothing is holding it alive (even pending for finalization is not allowed)
+            /// This include the work for scanning the sync blocks, so this value could indicate sync block usage issues.
+            /// This happens for any type of GC, and it should not be zero for all GCs.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            MarkLongWeak,
+
+            /// <summary>
+            /// This field records the time spent for planning.
+            /// Planning refers to the activity of determining where the contigous block of life memory (aka plugs) are and where they should be
+            /// relocated if we decided to compact
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            Plan,
+
+            /// <summary>
+            /// This fields records the time spent for relocating.
+            /// Relocation refers to the activity of changing all the pointers to objects to their new location.
+            ///
+            /// This happen only for blocking compacting GC.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            Relocate,
+
+            /// <summary>
+            /// This fields records the time spent for compacting.
+            /// Compacting refers to the activity of copying the ojbects from their old location to new location.
+            ///
+            /// This happen only for blocking compacting GC.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            Compact,
+
+            /// <summary>
+            /// This field records the time spent for sweeping.
+            /// Sweeping refers to the activity of inserting the free memory blocks into the size seggregated free
+            /// list if the corresponding generation.
+            ///
+            /// This happen only for blocking sweeping GC.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            Sweep,
+        };
+
+        /// <summary>
+        /// In .NET 6+, we have a poor man's profiler that capture the timing information
+        /// for various phases in GC. This provides a lightweight way to know how much time
+        /// is spent on each phase of the GC.
```

Comment:
```suggestion
        /// A lightweight way to know how much time is spent on each phase of the GC.
        /// Available in .NET 6+, 
```

Reply:
I would also change all the "Available for .NET 6+" to "Available in .NET 6+" from above.